graylog2-web-interface/src/components/users/UserCreate/TimeoutFormGroup.tsx

```diff
@@ -24,9 +23,6 @@ const TimeoutFormGroup = () => (
   <Field name="session_timeout_ms">
     {({ field: { name, value, onChange } }) => (
       <TimeoutInput value={value}
-                    labelSize={3}
-                    controlSize={9}
-                    name={name}
```

Comment:
Why are these removed?

Reply:
Because they are not used anymore in TimeoutInput.